lib/xiaomi.js

```diff
@@ -518,6 +518,17 @@ const numericAttributes2Payload = (msg, meta, model, options, dataObject) => {
                 payload.battery = value;
             }
             break;
+        case '1057':
+            if (['ZNCLBL01LM'].includes(model.model)) {
+                const lookup = {
+                    0: 'closing',
+                    1: 'opening',
+                    2: 'stop',
+                };
+                payload.motor_state = lookup[value];
```

Comment:
I think you need to take into consideration the value of the `invert_cover` option, as it was done in _case '1028'_:

https://github.com/Koenkk/zigbee-herdsman-converters/blob/d634a14cdc1bbb9ed0887df604dd884d1f0e0c4c/lib/xiaomi.js#L504-L510